manifest.json

```diff
@@ -496,9 +496,9 @@
         "targetDirPrefix": "ui-codemirror-"
       },
       "uiSortable": {
-        "version": "0.17.1",
+        "version": "0.19.0",
```

Comment:
Just to check, how does this get imported/used? Shouldn't we be changing something in a .ts file?

Reply:
Hi @seanlip, this gets combined in `generated/third_party.js` and is used from there.
I had planned to remove it from `generated/third_party.js` and apply it externally only to the places where ui-sortable is used, but I believe Yash is taking care of this lib.